lib/cisco_node_utils/bgp_af.rb

```diff
@@ -244,7 +243,11 @@ def default_advertise_l2vpn_evpn
 
     # dampen_igp_metric
     def dampen_igp_metric
-      config_get('bgp_af', 'dampen_igp_metric', @get_args)
+      match = config_get('bgp_af', 'dampen_igp_metric', @get_args)
```

Comment:
Clearly there is some default value (since you are returning it on the next line!) What is the default on N7k? Why can't you just provide a different default_value in the YAML for N7k rather than making this change?